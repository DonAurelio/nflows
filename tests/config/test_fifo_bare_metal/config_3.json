{
    "dag_file": "./tests/workflows/test_fifo_bare_metal/config_3.dot",
    "scheduler_type": "fifo",
    "mapper_type": "bare_metal",

    "core_avail_mask": "0x1",
    "flops_per_cycle": 32,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1000000000,

    "distance_matrices": {
        "latency": "./tests/system/test_fifo_bare_metal/3_lat.txt",
        "bandwidth": "./tests/system/test_fifo_bare_metal/3_bw.txt"
    },

    "log_base_name": "./tests/output/test_fifo_bare_metal/config_3",
    "log_date_format": ""
}
