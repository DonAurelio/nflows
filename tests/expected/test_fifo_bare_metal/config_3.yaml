exec_name_total_offsets:
  Task_3:
  Task_4:
  Task_2:
  Task_5:
  Task_1:
